DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift

```diff
@@ -91,6 +91,43 @@ class CalculatorVC: BaseViewController {
         accomplishedStepExamTF.tintColor = UIColor.primary
     }
     
+    func saveTextfieldData(textFieldName: textFieldName, data: String) {
```

Comment:
How about naming the method like `func saveTextFieldData(_ name: textFieldName, data: String)` ?